### ISSUE TITLE
Docker engine not running after update

### ISSUE THREAD

**Issue by dapalmi** [docker/for-win#1183]
### Expected behavior
Docker commands responding

### Actual behavior
Error message when executing docker commands in PowerShell

### Information

After updating to newest version I get following error message when trying to execute any docker command:
error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.32/images/json:open //./pipe/docker_engine: Das System kann die angegebene Datei nicht finden. In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running.

I already tried following fix suggestion: https://success.docker.com/KBase/Docker_for_Windows_fails_with_a_daemon_not_running_message

I also tried to uninstall and reinstall Docker without success.

### Steps to reproduce the behavior

  1. Updating to newest version (17.09.0-ce-win33) on Windows 10 Pro (Version: 1703, Build: 15063.632)

**Comment by IanHot** [comment]
I had the same issue and it turned out that the install had reset docker to Linux containers. Hope this helps

**Comment by dapalmi** [comment]
The problem is when starting Docker the whale is not even showing in the menu bar. So I have no way to change the settings.

**Comment by dapalmi** [comment]
Still having an issue. Newest version still does not work. Also since the update the VM disappeared from Hyper-V. I guess that is why I cannot start Docker. Any news here? Why does it take so long? Can you please provide a link to an older version before the catastrophic update so at least I can work?

**Comment by IanHot** [comment]
Have you modified the daemon.json file. I have found that any thing other than the standard set seems to stop the docker service dead?

**Comment by dapalmi** [comment]
No. I haven’t. But I saw that dozens of others have the same problem with Hyper-V after updating. Since Docker is not willing to roll back or to provide an link to an older version it seems like I have to wait until they fix this bug.

**Comment by kbeask** [comment]
I recently have the same issue in 2 of my computers (I installed the Docker Toolbox):
**Computer 1:** Windows 8.1  
**Computer 2:** Windows 10 (Home edition)
After the installation, everytime I run the bash, powershell... I get this result:
`$ docker info`
`error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.35/info: open                                                                                                                                                              //./pipe/docker_engine: El sistema no puede encontrar el archivo especificado.                                                                                                                                                              In the default daemon configuration on Windows, the docker client must be run el                                                                                                                                                             evated to connect. This error may also indicate that the docker daemon is not ru                                                                                                                                                             nning.`
So, I went to the directory where I installed Docker and double click on "start.sh" file. Then Docker works fine (for both of my computers), but is a pain in the neck to run Docker everytime with this "start.sh" file. I still have to check what's going wrong here... but I hope this 'solution' helps you guys!
